sql/src/main/codegen/config.fmpp

```diff
@@ -51,12 +51,15 @@ data: {
     # List of additional classes and packages to import.
     # Example. "org.apache.calcite.sql.*", "java.util.List".
     imports: [
+      "java.util.List"
       "org.apache.calcite.sql.SqlNode"
       "org.apache.calcite.sql.SqlInsert"
+      "org.apache.druid.java.util.common.Intervals"
```

Comment:
where is this used? similar for joda Interval import

Reply:
Removed